cmd/backend/backend.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 const BackendDir string = "./training"
+const AwsRegion = "us-west-2"
```

Comment:
Is this the convention for constants in go?

Reply:
yep